### PR TITLE
Edaggers: Actually works edition

### PR DESCRIPTION
**Pull request by theOperand**
you know the drill, except now they have 18 force instead of 15 and fancy attack verbs.

### CODE REVIEW

**Comment by Chocobro** [comment]
:+1: for that dang branch name

**Comment by Miauw62** [comment]
alright, conflicts resolved

**Comment by Miauw62** [comment]
lmao where did the uplink part go.

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -48,13 +49,17 @@
 	if(!istype(M))
 		return
 
-	if(M.can_inject(user, 1))
-		user << "<span class='warning'>You stab [M] with the pen.</span>"
-		if(!stealth)
-			M << "<span class='danger'>You feel a tiny prick!</span>"
-		. = 1
+	if(!force)
```
Comment (Miauw62) [inline code/modules/paperwork/pen.dm]: This is ugly, but idk how to do it in a non-ugly way. should i add an extra argument and then use that to skip it only in the edagger child? (e.g. call ..(M, user, stealth, extra) in edagger/attack(), and have extra just cause it to call ..())

File: code/modules/paperwork/pen.dm [hunk 2]
```diff
@@ -77,4 +82,43 @@
...
+		w_class = initial(w_class)
+		name = initial(name)
+		hitsound = initial(hitsound)
+		embed_chance = initial(embed_chance)
+		throwforce = initial(throwforce)
+		playsound(user, 'sound/weapons/saberoff.ogg', 5, 1)
+		user << "<span class='warning'>[src] can now be concealed.</span>"
+	else
+		on = 1
+		force = 18
```
Comment (phil235) [inline code/modules/paperwork/pen.dm]: I thought you said 20.

**Comment by Stenthus** [comment]
these comparisons are made against an unarmored, uninjured, un-healing chem'd target

At 15 damage it takes (as lumi noted last time) 7 hits, the same as a lit welder to crit (note I don't actually remember the last time I've seen someone go on a welder rampage, or even defend themselves with one against non-blob/alien enemies.)

At 18 damage they are comparable to stealthy, easier to use spears and require 6 hits to crit an uninjured unarmored dude

(3 tc)At 20 damage, this energy dagger is it's 5 hits to crit

Higher damage alternatives:
(limited)wielded fire axe: 24 damage, will take 5 to crit but often 4 since let's be real it's pretty easy to have taken 4 extra damage at some point
(8 tc)energy sword: 30 damage, 4 hits to crit
(16 tc)double esword: 34 damage, 3 hits to crit

There is an important caveat to these damage comparisons, though, and that's that this is very likely to be used with an ebow, which is 15 toxin/ 5 second stun
in this situation, it:
Requires one less hit to crit than a welder, one more than an esword, and the same as a wielded spear

After thinking about this a lot I find 20 damage reasonable as it's also the same as the switchblade and 3 tc isn't that much. the utility of being able to hide it in a pda is also very helpful as confiscating/checking pda pens is rare I find.

I can easily see this replacing the esword as the go to weapon in combination with the ebow while still allowing for 4 tc of extra purchases

File: code/modules/paperwork/pen.dm (hunk 2 above)
Comment (Miauw62) [inline code/modules/paperwork/pen.dm]: Yeah, I changed it to 18 and forgot to change the title.

**Comment by Miauw62** [comment]
Does it make a big differene if its 18 instead of 20?

**Comment by Stenthus** [comment]
if it's 18, it takes 6 hits to crit someone unarmored/at full hp
if it's 20, it takes 5

with an ebow shot included, it's 5 shots to crit someone either way

personally i'd prefer it as 20 since then it's as good as a switchblade but can be hidden better. spears, which are relatively easy to obtain by anyone with access to wires, do 18 damage too.

if you're assuming going into crit is essentially like dying (which it is most of the time) then 20 damage is significantly better in a straight up full health-crit fight to the death since it takes 1 less hit

**Comment by PKPenguin321** [comment]
18 is fine though because this shouldn't be as good as the switchblade or the esword

**Comment by Fox-McCloud** [comment]
Going to agree this should probably be 18.

20 is pushing it for a 3 TC item.

**Comment by Robustin** [comment]
20 Is fine, esword aren't a good reference since they're overpriced as-is.

I'd rather traitor items start out strong and then tone it down if OP, rather than the current issue of having a huge list of undesirable weapons/gadgets.

**Pull request merged**